1591: Remove import pipeline nuernberg

seluianova opened the pull request:
### Short Description
Remove import pipeline nuernberg 
_(decided to move this to a separate PR, assuming this would be easier to test and review)_

### Proposed Changes
- remove nuernberg pipeline (and some other unused entities that I noticed)

### Side Effects
no?

### Testing
1. Check that the csv import for Nürnberg is still working
2. Check that the "Import data" job is still working for Ehrenamtskarte Bayern

### Resolved Issues
Fixes: #1591

seluianova commented on README.md:
```diff
@@ -22,7 +22,6 @@
 * [Protobuf Code Generation](./docs/protobuf-generation.md)
 * [Migrations](./docs/migrations.md)
 * [Deep Linking](./docs/deep-linking.md)
-* [Socialpass Stores import](./docs/sozialpass-stores-import.md)
```
I assume this can be deleted and we didn't want to actualize it?

seluianova commented on backend/src/main/resources/import/Akzeptanzpartner-daten.csv:
we don't use this anyhow anymore right?

f1sh1918 replied:
no we have example store import data in `administration/resources`

steffenkleinle reviewed (approved):
LGTM

f1sh1918 commented on backend/src/main/resources/config/config.yml:
```diff
@@ -38,8 +38,7 @@ projects:
   - id: nuernberg.sozialpass.app
-    importUrl: https://data.entitlementcard.app/nuernberg-akzeptanzstellen.csv
-    pipelineName: SozialpassNuernberg
+    importUrl: ""
```
We have to check if the pipeline name is still in the salt repo used just to ensure that the server Import will not fail because the pipeline is not existing anymore after this change

f1sh1918 replied:
please also remove the `pipelineName` for koblenz in this file

seluianova replied:
https://git.tuerantuer.org/DF/salt/pulls/326

bahaaTuffaha reviewed (approved):
Imported stores & cards to nuernberg and tested card imports for bayern.
All sounds good 👍️

f1sh1918 commented:
Looks quite good already.
What is missing to cleanup is the `csvWriter` config and the related code f.e. `EhrenamskarteBayern.kt`
That is also not needed anymore

Please also remove the `pipelineName` from the `config.test.yml`

seluianova commented:
@f1sh1918 thanks!

- removed csvWrites
- removed pipeline name for Koblenz and nuernberg from configs
- opened PR in the salt repo: https://git.tuerantuer.org/DF/salt/pulls/326

f1sh1918 reviewed (approved):
Looks good 👍 
Please do not merge before salt pr was merged and deployed!
